packages/deployment/test/test_kes.js

```diff
@@ -5,17 +5,19 @@
 const sinon = require('sinon');
 const test = require('ava');
 const get = require('lodash.get');
+const clonedeep = require('lodash.clonedeep');
 
 const configFixture = require('./fixtures/config.json');
 const aliasFixture = require('./fixtures/aliases.json');
 const UpdatedKes = require('../lib/kes');
 
 test.beforeEach((t) => {
-  t.context.kes = new UpdatedKes(configFixture);
+  const config = clonedeep(configFixture);
```

Comment:
Kes mutates it's configuration object -- without this, we were causing havoc, particularly a false positive, in later tests l due to the configuration being modified in tests.